src/System.Management.Automation/CoreCLR/CorePsPlatform.cs

```diff
@@ -475,11 +475,6 @@ internal static bool NonWindowsIsHardLink(FileSystemInfo fileInfo)
             return Unix.IsHardLink(fileInfo);
         }
 
-        internal static bool NonWindowsIsSymLink(FileSystemInfo fileInfo)
-        {
-            return Unix.NativeMethods.IsSymLink(fileInfo.FullName);
```

Comment:
Can we add a comment to the native code as to when this call was removed?

Reply:
I could but I don't understand why we need this - in last days we removed some P/Invokes 
and earlier a few more without such comments. Perhaps it is more useful to open a tracking issue in PowerShell Native repo.

Reply:
A tracking issue would work, but the issue is likely to be open for a long time.